seshdash/views.py

```diff
@@ -690,4 +691,64 @@ def historical_data(request):
         context_dict['active_site'] = active_site
         context_dict['sort_keys'] = sort_data_dict.keys()
         context_dict['sort_dict'] = sort_data_dict
+        print sort_data_dict
         return render(request, 'seshdash/historical-data.html', context_dict);
+
+def influx(request):
+    client = InfluxDBClient()
+    client = InfluxDBClient(database='test_db')
+    measurements = client.query('show measurements')
+    measures = measurements.raw
+    measure = measures['series']
+    data_db = measure[0]
+    data = data_db['values']
+    #print measure
+    #print data_db
+    #print data
+    return  HttpResponse(json.dumps(data))
+
+def influxvalues(request):
+    results = {}
```

Comment:
You should fir check if the request is a POST. Else this will give an error if someone does a GET request o n this endpoint. Also there should be @login_required decorator as well on both these functions.